pkg/network/protocols/postgres/model_linux.go

```diff
@@ -73,6 +73,21 @@ func (e *EventWrapper) Operation() Operation {
 	return e.operation
 }
 
+// extractParameters returns the string following the command
+func (e *EventWrapper) extractParameters() string {
+	b := getFragment(&e.Tx)
+	idxParam := bytes.IndexByte(b, ' ') // trim the string to a space, it will give the parameter
+	if idxParam == -1 {
+		return ""
+	}
+	idxParam++
+	idxEnd := bytes.IndexByte(b, '\x00') // trim trailing nulls
```

Comment:
This could maybe search in `b[idxParam:]` to avoid searching the first part again?

Reply:
done